OpenRA.Mods.Common/Traits/Air/AttackPlane.cs

```diff
@@ -37,7 +37,7 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new FlyAttack(self, newTarget);
 		}
 
-		protected override bool CanAttack(Actor self, Target target)
+		public override bool CanAttack(Actor self, Target target)
```

Comment:
I'm not really happy with the idea of mixing an architectural rework (which I personally don't think should even be changed) like this in with new features.